analyzers/tests/SonarAnalyzer.TestFramework/Verification/DiagnosticVerifier.cs

```diff
@@ -141,59 +141,35 @@ ReportDiagnostic Severity(string id)
             }
         }
 
-        private static void CompareActualToExpected(string languageVersion, Diagnostic[] diagnostics, FileIssueLocations[] expectedIssuesPerFile, bool compareIdToMessage)
+        private static void CompareActualToExpected(Diagnostic[] diagnostics, CompilationIssues expected, bool compareIdToMessage)
```

Comment:
The parameter `compareIdToMessage` is useless (for now).
It's part of the next PR, right?

Reply:
I don't know yet. I will either use it or remove it in #8601